Source/Core/DolphinQt/Config/NewPatchDialog.cpp

```diff
@@ -117,54 +141,40 @@ QGroupBox* NewPatchDialog::CreateEntry(PatchEngine::PatchEntry& entry)
 
   auto* offset = new QLineEdit;
   auto* value = new QLineEdit;
+  auto* comparand = new QLineEdit;
 
   m_edits.push_back(offset);
   m_edits.push_back(value);
+  m_edits.push_back(comparand);
+
+  auto* conditional = new QCheckBox(tr("Conditional"));
+  auto* comparand_label = new QLabel(tr("Comparand:"));
 
   auto* layout = new QGridLayout;
   layout->addWidget(type, 0, 0, 1, -1);
   layout->addWidget(new QLabel(tr("Offset:")), 1, 0);
   layout->addWidget(offset, 1, 1);
   layout->addWidget(new QLabel(tr("Value:")), 2, 0);
   layout->addWidget(value, 2, 1);
-  layout->addWidget(remove, 3, 0, 1, -1);
+  layout->addWidget(conditional, 3, 0, 1, -1);
+  layout->addWidget(comparand_label, 4, 0);
+  layout->addWidget(comparand, 4, 1);
+  layout->addWidget(remove, 5, 0, 1, -1);
   box->setLayout(layout);
 
-  connect(offset, qOverload<const QString&>(&QLineEdit::textEdited),
-          [&entry, offset](const QString& text) {
```

Comment:
It just occurred to me; storing references to entries is very unsafe, since they are elements of a vector that can be allocated! I haven't made this problem worse, but I think I'll have a go at fixing it.